sdk/storage/test-resources.json

```diff
@@ -17,6 +17,10 @@
             "metadata": {
                 "description": "The principal to assign the role to. This is application object id."
             }
+        },
+        "testEndpointSuffix": {
+            "type": "string",
+            "defaultValue": "storage.azure.net"
```

Comment:
@benbp Since this endpoint suffix value `storage.azure.net` only applies to private preview subscriptions, is there a way to set this as we do with the other parameters?